bond/bond.go

```diff
@@ -455,6 +455,10 @@ func cmdDel(args *skel.CmdArgs) error {
 		return fmt.Errorf("Failed to deattached links from bond, error: %+v", err)
 	}
 
+	if err = util.HandleMacDuplicates(linkObjectsToDeattach, netNsHandle); err!=nil {
```

Comment:
```suggestion
	if err = util.HandleMacDuplicates(linkObjectsToDeattach, netNsHandle); err != nil {
```

Just for consistency with the rest of the file.

Reply:
Done